posthog/temporal/data_imports/pipelines/doit/source.py

```diff
@@ -83,8 +89,25 @@ def doit_source(
     report_id = selected_reports[0]
 
     def get_rows(report_id: str):
+        request_uri = f"https://api.doit.com/analytics/v1/reports/{report_id}"
+
+        if should_use_incremental_field and db_incremental_field_last_value is not None:
+            if isinstance(db_incremental_field_last_value, datetime.date):
+                start = db_incremental_field_last_value.strftime("%Y-%m-%d")
+            elif isinstance(db_incremental_field_last_value, datetime.datetime):
+                start = db_incremental_field_last_value.strftime("%Y-%m-%d")
+            elif isinstance(db_incremental_field_last_value, str):
+                date = parser.parse(db_incremental_field_last_value)
+                start = date.strftime("%Y-%m-%d")
```

Comment:
Since this is an `Optional[Any]` type, shouldn't there be a catch all in the event that none of the expected types are here for `db_incremental_field_last_value`? Also, if I'm reading this correctly, `start` would be undefined in this case.

Reply:
Good spot, have fixed - thanks!